lib/openregister.rb

```diff
@@ -15,6 +15,10 @@ def _records
     OpenRegister::records_for register.to_sym, try(:_base_url_or_phase)
   end
 
+  def _entries
```

Comment:
The `_` eludes to this being local, but I don't see it being used anywhere but in the test. What do you intend on doing with this?